### DISCUSSION TITLE
Equipo 15 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#22] [Proyecto Final]
Hola Equipo 15,
Aquí está el hilo para coordinar entre ustedes, charlar con el Equipo de Educación, recibir retroalimentación sobre cada etapa.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
@ayekn @Antuuu08 @AmbarPranahuti
Formarán el equipo 15. Cualquier consulta, aquí ando!
Kasturi

**Reply by ayekn** [reply]
@Ayegkn Dejo mi nombre de usuario para contactarse conmigo por telegram o wts al número 03513314703 y hablar del proyecto

**Reply by Estefania-369** [reply]
Hola @ayekn Ayelen buen dia!
Salud Medicinal, soy Estefania Rivas, escribo para compartirte retroalimentacion respecto a la etapa O de tu proyecto final. 

Primeramente, me agradaria felicitarte por tu trabajo y que tomaras la desicion de continuar en el curso, es maravilloso, mis mejores deseos! :D 

Continuemos con la retro:

- Esta muy interesante la problematica que has seleccionado, es algo que tambien es fundamental tomarle importancia, seguramente le daras un gran desarrollo. 
- Me parecen bien planteadas tus preguntas en relacion a la problematica que has elegido.
- La presentacion y como te diriges al entevistado me parece super bien. 
- No es obligatorio, pero estaría agradable que agregaras una pregunta para romper el hielo al inicio, a modo sugerencia.
- Tambien a modo sugerencia, podrias agregar un pequeño dialogo al final de la entrevista, de despedida en donde puedes agradecerle al entrevistado por su tiempo, por compartir sus ideas, experiencias a atraves de la entrevista.
- Lo que llevas hasta el momento y como lo has planteado me parece perfecto!

Buen trabajo Ayelen

Si tienes alguna duda, sugerencia, estamos para apoyarte
Mucho exito

Saludos
Estefania
:D

**Reply by KasturiSShah** [reply]
Gracias @Estefania-369 por revisar la etapa y por compartir retroalimentación! @ayekn, Fanny es moderadora dentro del Equipo de Educación de Patrones Hermosos. Ella revisará tus etapas a lo largo del curso 😄 
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Antuuu08 @AmbarPranahuti

Espero estén muy bien. Escribo para actualizarlas que ahora formarán el equipo 17. Aquí está su carpeta grupal en el repositorio: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-17

¿Cómo andan con el proyecto? ¿Podríamos apoyarles en cualquier cosa? Tienen prórroga hasta este domingo para la etapa 0. Si necesitan más tiempo, me avisan.

No duden en aclarar cualquier pregunta sobre el proyecto final por este medio o en una asesoría.

Mucha suerte y saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @ayekn 
Gracias por entregar la etapa 0 del proyecto final. Justo vi la carpeta actualizada. Estaré revisando el protocolo este fin de semana y te escribiré por este medio con retroalimentación. Tienes prórroga para la etapa 1 del proyecto final hasta el domingo próximo. 😄 
Saludos,
Kasturi

**Comment by AmbarPranahuti** [comment]
Hola!!! Ya elegiste la temática del proyecto?

**Comment by Estefania-369** [comment]
**Hola @ayekn !!!**
Muchas gracias por compartir avance de la etapa 1.
El objetivo que has planteado para el desarrollo de tu app me parece excelente, ¡me encantara verlo teminado!  
Si tienes alguna duda para finalizar con la etapa 1, con gusto te respondere, puedes etiquetarme haciendo una mencion con mi nombre de usuario.

Mucho exito
bonito dia 
Fanny
:D

**Comment by ayekn** [comment]
no entiendo que me falta para terminar de la 1? yo ya estaba por hacer la 2
parte, lo hago?

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by Estefania-369** [reply]
Hola @ayekn buenas tardes!
Disculpa la tardanza en responder 
No te preocupes, en la etapa 1 unicamente seria agregar una foto o una captura de pantalla de la "Sintetización de Apuntes Tomados durante tu Entrevista", por ejemplo podrias usar Jamboard para la sintesis de las respuestas de tus entrevistados. 
Puedes continuar con la etapa 2 sin problema. 
Estoy a la orden y mucho exito 
Saludos 
:D

**Comment by Estefania-369** [comment]
Hola @ayekn ! :D 

Gusto en saludarte, gracias por entregar la etapa 2 del proyecto final, así como tu dedicación en el desarrollo del mismo. Te escribo para compartir retroalimentación.

Me parece que va muy bien el desarrollo del proyecto, lo primero que presentas utilizando la técnica 8s (ocho loco) considero que son interesantes cada una de tus ideas, están enfocadas en el planteamiento del problema que deseas dar solución y cubren cada uno de los diferentes escenarios que nos podemos llegar a encontrar al momento de tener un tratamiento médico.

También abarcas puntos clave en la lluvia de ideas, que te servirán para llegar a desarrollar una eficaz app.  
La parte donde utilizas el método SCAMPER, me parece bien implementada, ya que dividiste los métodos utilizados, para reducir tus ideas y llegar a las más factibles.

Como sugerencia en la parte donde implementas “COMBINAR”, para que sea más entendible y visual, podrías colocar un post-it donde escribas como quedo el resultado final al combinar tus 2 opciones. Algo así como lo que hiciste con lo de “MODIFICAR”. 
Está bien distribuido, ya que has utilizado colores diferentes en los post-it para una mejor clasificación. 
Me agrada tu trabajo, se ve que ha quedado entendible como se utiliza este tipo de métodos para generar ideas, y así llegar a un resultado final. 

Te felicito por tu trabajo y dedicación. 👍 
Si tienes alguna duda, estamos para apoyarte.

Un abrazo
Estefania

**Comment by Estefania-369** [comment]
Hola @ayekn ! :D 

Gusto en saludarte nuevamente, gracias por entregar la etapa 3 del proyecto final, así como tu dedicación en el desarrollo del mismo. Te escribo para compartir retroalimentación de esta etapa.

Me parece excelente como estas desarrollando el prototipo low-fidelity y como lo explicas en el video, se puede percibir que esta enfocado en el principal Objetivo. 

Es increíble el desarrollo de cada una de las opciones que llevara la app, y me parece super bien que primero lo presentaras con dibujos, y después lo expongas en Wireframe en figma.  Esto facilitara recibir retroalimentacion de personas que se los muestres y puedas iterar tu app. 

La primera parte que presentas, está muy bien planteada y diseñada, para un registro, acceder datos personales, etc., en la app. 

En la parte de datos personales, me gustaría hacerte una sugerencia y pregunta: Crees que sería relevante incluir también datos que incluyan como el peso, estatura, cuáles son sus alergias si es que las llegara a tener, si es alérgico a un tipo de medicamento, tipo de sangre, si ha tenido operaciones, etc., datos que nos puede llegar hacer el médico, y ¿tenerlos como un registro en la app? Siendo parte del historial médico.

La segunda parte que presentas, también me parece que esta muy bien diseñada, y se ve intuitiva las opciones. 
Donde colocan las pastillas y sus gramos, así como las alarmas, me gusta como las vinculas para que sea simple de utilizar. Esta bien detallada ya que hasta colocaste la opción de agregar nuevas pastillas. 

Me encanta también la parte del calendario, se ve muy visual, dinámico y fácil de utilizar, me agrada que colocaste también un icono para identificar las citas médicas, que se vea visual, así como marcar en rojo a modo de alarma que ya está por terminaste el medicamento.

Me encanto como estas desarrollando la app, el avance esta cumpliendo con tu objetivo y la solución a la problemática. 

Se que vas a seguir iterando, para ir mejorando, modificando tu app.

Solo me agradaría sugerir, si has considerado algún botón, u opción para regresar a las otras opciones o pantallas, por ejemplo, si estoy en calendario y deseo regresar al menú principal, ¿cómo regresaría?

Otro punto de vista es que si has planteado como se verían las notificaciones de la app en el teléfono, pongamos el caso siguiente: me va a mostrar una pantalla completa, mientras este usando otra app como whatsapp, por ejemplo, con el calendario, para recordarme la cita con el médico o que ya se va a terminar el medicamento. O se me presentara una pequeña notificación con letra para acceder a la app, y ver el calendario. 

Por el momento, seria todo, vas de maravilla con el progreso de tu proyecto. 👍 

Si tienes alguna duda, estamos para apoyarte.
Espero que estes teniendo una agradable semana, mucho éxito y mis mejores deseos.
Un abrazo
Estefania

**Comment by ayekn** [comment]
Buenas, disculpen si no respondí ante, la verdad que tomé sus consejos y ya
finalice mi high fidelity, diculpen que lo haya terminado tarde, pero estoy
sola en el proyecto, no me gustaría perder la oportunidad de mostrar a
todas por lo menos un avance de lo q hice. Espero sus respuestas, muchas
gracias

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Estefania-369** [comment]
Hola @ayekn !
No te preocupes, que gusto que has avanzado con tu proyecto y lo estas mejorando 
Me alegra que vas bien con tu prototipo!
Si tienes alguna duda quedo a la orden 
Sera emocionante ver tu proyecto 
Mucho exito!
Linda tarde 
Un abrazo

**Reply by KasturiSShah** [reply]
De acuerdo con @Estefania-369!

No te preocupes @ayekn, sería excelente mostrar los avances que has hecho. _No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo. _

También, te comento que la etapa 5 es corta (cosa de 30 minutos) y no necesita que las etapas 3 y 4 sean completadas para entregarlo. Entonces, una opción sería entregar la etapa 5 y luego completar las demás etapas. Instrucciones para la etapa 5 están disponibles aquí en tu carpeta grupal: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-15/etapa-5/README.md

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar su día asignado al otro día. (Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing))

Mucha suerte, que sigas adelante, y hasta pronto

Kasturi

**Comment by Estefania-369** [comment]
Hola @ayekn ! :D 

Me da gusto saludarte, gracias por entregar la etapa 4 del proyecto final, así como todo el trabajo en el desarrollo del mismo. Te escribo para compartir retroalimentación.

**OJO:** Dado las alturas del curso, las recomendaciones, sugerencias, que se pudieran presentar, no es obligatorio realizar cambios al proyecto, antes de tu presentación final en la clausura. 

Esto para evitar estrés o presión por el corto tiempo. Si en dado caso que tu desees aplicarlos, esta bien, es perfecto. 
Solamente a modo de recordatorio: No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo.

Dicho esto, procedo con la retro.

¡Me encantaron los colores que has utilizado en tu presentación, así como en toda la app! El logo de “SaludMedicinal”, esta increíble, has cuidado muy bien cada detalle, la cajita con las pastillas, la capsula, las imágenes que están dentro, los colores, uff super todo, proyectas excelente lo que se trata tu proyecto. ¡Felicitaciones! 

**Interfaz 1:**

Las cápsulas kawaii (tierno, cute) están increíbles, le das un toque amigable, dinámico, agradable, original y divertido a toda la app. 
Me ha sorprendido como has plasmado a detalle absolutamente todo lo que llevará tu app, está claro el inicio de sesión, ofreces la opción de inicio por Google, es fantástico ya que eso lo hace simple, tomando en cuenta que tus usuarios potenciales serían los adultos mayores. 

**Interfaz 2:**

Se comprende el menú, es resumido y tiene buen tamaño la letra también. Me parece muy bien el icono de la flecha regresándose, ya que si se entiende su función. La pantalla final de datos guardados, perfecta, el ícono de una pequeña casa en la parte superior parece ser que es para regresar al menú. (Espero que comprendiera bien esta parte del icono).

**Interfaz 3:**

Me agrada que indicas como se iluminara la opción que el usuario seleccione (color violeta), así como el teclado para escribir. También la flecha de adelante esta excelente. Esta clara esta parte. 
Solamente me queda una pequeña duda: El icono con las 3 barras, en la parte inferior izquierda, ¿qué es lo que desplegará? será el menú para seleccionar alguna opción e ir directamente, sin tener que regresarse al menú.

**Interfaz 4:**

Para ingresar los datos del doctor, esta claro y simple, se entiende bien incluso el icono de mas (+) que seria para agregar a mas doctores si es que los hubiera.
(Dato curioso y random: es interesante que te guste el beisbol, lo comento por el nombre que has agregado, me llamo la atención y lo investigue, ¡muy genial!)
¡Una vez mas recalco que me fascinan las imágenes kawaii mostrando cada una de las opciones!

**Interfaz 5:**

Agregar pastillas parte 1: Se entiende todo, incluso que colocaste el signo de mas (+) para agregar los gramos de la pastilla y la alarma.

**Interfaz 6:**

Para agregar la alarma esta simple de comprender, debo compartir que, en mi caso, la parte donde está la opción de vibrate (vibrar) tarde un poco en comprender los colores, cual seria el de activado y el de apagado. El color morado desactivado y el rosa activado. Por ejemplo, regularmente estamos acostumbrados a ver el modo apagado con un color gris, y el de activado con un color más vivo.  Ojo, no quiero decir que este mal, simplemente comparto mi percepción o como fue mi experiencia, pero me agrada mucho el color morado combinado con el rosa, eso sí debo de enfatizar. 
Se entiende perfecto que, en los días de la semana, los que están en color negro, son los días que no selecciono para que se active la alarma, ósea días que no le toca tomar el medicamentó. 

**Interfaz 7:**

Queda muy claro el menú donde se muestran las pastillas agregadas, esta magnifico las imágenes de las pastillas ya que así ayuda de manera visual a identificarlas y diferenciarlas, se entiende los iconos de agregar y borrar. 

**Interfaz 8:**
Me agrada la opción del Buscador, es fácil de utilizar y resulta útil.

**Interfaz 9:**
Esta clara la parte de cómo se vería al final una vez agregadas las pastillas, me agrada que el color del indicador de la alarma es diferente, para distinguir. 

**Interfaz 10:**
Esta increíble el diseño del calendario, como has usado distintos colores para diferenciar las opciones como agregar citas, pastillas, etc., También la opción de agregar al calendario un recordatorio de las citas o pastillas. 
La parte del icono de un cuadrado con una paloma o check, a mi interpretación (se que lo vas a explicar en tu presentación) es simplemente para seleccionar la pastilla que deseas ver en el calendario con la información que le corresponde. 

**Interfaz 11:**
Me agrada muchísimo que expliques como pueden encontrar o diferenciar al doctor, y así seleccionarlo para agregarlo a la cita en el calendario. 
Se ve simple y entendible el calendario. 

 **Interfaz 12:**
¡Me encanta el resultado final, se entiende cada uno de los iconos con sus respectivos colores!
Te felicito muchísimo por todo el trabajó, se siente y percibe toda la energía, tiempo, entrega en tu proyecto, como estar iterando con el usuario, y estar en contantemente cambio ha sido de mucho apoyo para ti, y así seguir renovando y mejorando tu app. 

Para no hacerlo tan extensa la retro, no me queda más que agradecerte a ti en compartir tu hermoso proyecto.
¡Sera emocionante que lo presentes en la clausura y lo puedas compartir con todxs!
 
Mucho éxito :+1: 
Un abrazo
Estefanía

**Comment by Estefania-369** [comment]
Hola @ayekn ! :D 

Me da gusto saludarte nuevamente, gracias por entregar la etapa 5 y ultima del proyecto final, (¡se pasó super rápido el tiempo!) me alegro de haber compartido este recorrido contigo, en donde aprendimos juntas. 

Te escribo para las observaciones de esta etapa.
Me agrada las tareas que has tomado en cuenta, esto a consideración de tus requerimientos, y observaciones que recibiste desde la persona entrevistada, así como a las sugerencias. 

Sera interesante y emocionante ver esas interfaces que vas agregar. 
Me agrada que decidas añadir fotos de las pastillas, esta es una buena forma también de que los usuarios puedan identificar sus pastillas, tomando la foto de cada una de las cajas o pastillas. Consideras todo tipo de detalles, sin duda una app muy completa.

En la tarea de poner que hacer, me parece que, si está a consideración, dependiendo de posibles iteraciones extras con más personas. Una idea que se me presento, considerando que mencionaste lo de la letra y que posiblemente los adultos mayores no alcancen a ver el texto, se me ocurrió compartirte que podrías incluso agregar un icono de una pequeña bocina, en donde lo seleccionen, y se escuche una voz que lea lo que dice el texto, o que explique de manera resumida lo que tienen que hacer en cada uno de los apartados del menú, como las asistentes, por ejemplo como la de Google, o que los personajes de la pastillas kawaii sean lo que pudieran hablar y explicar lo que ya mencione, algo como se ha visto en algunos juegos para teléfono. 
Esta podría ser una sugerencia, todo depende de ti, de tus usuarios, se trata de que estes mejorando, iterando una y otra vez. 

Me parece perfecto que mejores tus conocimientos sobre Figma, para seguir renovando tu proyecto y quede fluido como llegaste a mencionar. 

Ha sido un excelente Sprint, y es de mucha utilidad para poder observar y analizar las tareas y tener una mejor organización y avance en el prototipo/proyecto. 

¡Una vez mas te felicitó por todo tu trabajó!

Es asombroso, me encantara verlo funcionando
Se que seguirás mejorando, renovando hasta que logres incluso programarlo, desarrollarlo, mis mejores deseos. 

Te deseo una bonita semana, y nos vemos pronto en la clausura, ¡que emoción!
Saludos y un abrazo
Estefanía

PD: me gusto la canción de “mary on a cross”, incluso me puse a escuchar otra canción de ellos “square hammer” y también me agrado, mientras escribí tu retro las estuve escuchando, gracias por compartir, esta interesante el estilo de esa banda.

**Comment by ayekn** [comment]
Buenas pero no encuentro donde subir el video en Github que sería la parte 6

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by KasturiSShah** [reply]
Hola @ayekn 

Gracias por avisarnos. Justo he creado el archivo. Aquí está el archivo que tienes que rellenar para completar la etapa 6: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-15/etapa-6/README.md

Saludos,
Kasturi
